dialect.go

```diff
@@ -31,21 +36,15 @@ type Dialect interface {
 	ReplaceInto(src, dst string) error
 }
 
-var (
-	dialects = make(map[string]Dialect)
-)
-
-// RegisterDialect :
-func RegisterDialect(driver string, d Dialect) {
```

Comment:
I think with this `RegisterDialect` would be better, if anyone want to have their own Dialect instead of `MYSQL` or `POSTGRESQL`. But it's good to prevent they used unstable dialect, mean the dialect we enable to use is stable and usable.